pkg/dashboard/ttlcontroller/ttlcontroller.go

```diff
@@ -55,13 +52,15 @@ type TTLconfig struct {
 
 // NewController returns a new database ttl controller
 func NewController(
+	logger logr.Logger,
 	experiment core.ExperimentStore,
 	event core.EventStore,
 	schedule core.ScheduleStore,
 	workflow core.WorkflowStore,
 	ttlc *TTLconfig,
 ) *Controller {
 	return &Controller{
+		logger:     logger,
```

Comment:
This constructor is used with the "dependency injection framework" called fx. We still need to give it the name "ttlcontroller".

So we could not just feed the "constructor" to the fx(cmd/chaos-dashboard/main.go:94), we need to create another function to do similar things, including naming. Please take a look at https://github.com/chaos-mesh/chaos-mesh/pull/3035, using the similar solution.